index.html

```diff
@@ -0,0 +1,349 @@
+        don’t hestiate to contact me.</p>
+      <a class="get-myresume button" href="">Get My Resume</a>
+      <img class="abt-me-image" src="./assets/images/Group1.png" alt="">
+    </div>
+    <div class="about-me-desk">
+      <div class="story-abt-me">
+        <h1 class="h31-desk">About me</h1>
+        <p class="p31-desk">Hello I’m a software developer! I can help you build a product ,feature or
+          website Look through some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.</p>
+        <a class="get-myresume button" href="">Get My Resume</a>
+      </div>
+      <img class="abt-img-desk" src="./assets/deskimage/Groupme.png" alt="">
+    </div>
+
+    <ul class="lists">
+      <li class="Lan">Language
+        <ul class="lan-list">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </li>
+
+      <li class="Frame"> Framework
+        <ul class="Frame-list">
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </li>
+      <li class="Sk">Skills
+        <ul class="Sk-list">
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </li>
+    </ul>
+  </section>
+  <form class="upper-form" action="https://formspree.io/f/xoqrrprd" method="post">
+    <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+    <input type="text" maxlength="30" required placeholder="Full Name" class="full-name-m" name="usrname">
+    <input type="email" required placeholder="Email" class="email-m" name="usremail">
+    <textarea maxlength="500" id="msg" placeholder="Enter text here" class="messages-m" name="usrmessages"></textarea>
+    <input type="submit" value="Get it touch" class="touch-m" name="usrsubmit">
+  </form>
+
+  <form class="upper-form-desk" action="https://formspree.io/f/xoqrrprd" method="post">
+    <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch. </p>
+    <div class="inputs-form">
+      <div class="form-name-desk">
+        <input type="text" maxlength="30" required placeholder="Name" class="full-name-desk" name="usrname">
+        <input type="text" maxlength="30" id="lnm" required placeholder="Last Name" class="last-name-desk"
+          name="usrname">
+      </div>
+      <input type="email" required placeholder="Email" class="email-desk" name="usremail">
+      <textarea maxlength="500" id="msg" placeholder="Hello Solomon. I would like to get in touch with you..."
+        class="messages" name="usrmessages"></textarea>
+      <input type="submit" value="Get it touch" class="touch-d" name="usrsubmit">
+    </div>
```

Comment:
Kindly ensure your text and your form are at the same level as shown in the design.😉

design
<img width="483" alt="text" src="https://user-images.githubusercontent.com/64643995/220952925-b8d69ae5-1ea3-40ea-9238-4a78c63f5fdb.png">

yours
<img width="905" alt="textyirurs" src="https://user-images.githubusercontent.com/64643995/220953089-49b9309d-8b83-4231-a8b6-e0e0c718da6d.png">

also, consider the font properties and make your project similar to the design😉